setup.cfg

```diff
@@ -3,7 +3,7 @@
 ;;
 [metadata]
 name = fonzie
-version = 0.2.1
+version = 0.2.2
```

Comment:
It has not been released yet :grimacing:

Reply:
Whoops, just a first test that I forgot to undo ..